octoprint_discordremote/__init__.py

```diff
@@ -400,6 +423,11 @@ def notify_event(self, event_id, data=None):
         data['timespent'] = self.get_print_time_spent()
         data['eta'] = self.get_print_eta()
 
+        # Attempt to parse potential thumbnail
+        if event_id == "printing_started" and self.get_gcode_format(data["path"]):
+            self._logger.info("Print appears to have a thumbnail, trying to parse")
+            print_thumbnail = self.get_thumbnail_cura(data["path"])
```

Comment:
initialise print_thumbnail to None at the start of this function, otherwise it will cause unbound errors like:

```
Traceback (most recent call last):
  File "/home/cameron/Projects/OctoPrint-DiscordRemote-Dealman/testenv/lib/python3.10/site-packages/octoprint/plugin/__init__.py", line 273, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/home/cameron/Projects/OctoPrint-DiscordRemote-Dealman/testenv/lib/python3.10/site-packages/octoprint/util/__init__.py", line 1694, in wrapper
    return f(*args, **kwargs)
  File "/home/cameron/Projects/OctoPrint-DiscordRemote-Dealman/octoprint_discordremote/__init__.py", line 311, in on_event
    return self.notify_event("startup")
  File "/home/cameron/Projects/OctoPrint-DiscordRemote-Dealman/octoprint_discordremote/__init__.py", line 462, in notify_event
    return self.send_message(event_id, tmp_config["message"].format(**data), tmp_config["with_snapshot"], print_thumbnail)
UnboundLocalError: local variable 'print_thumbnail' referenced before assignment
```

Reply:
Something like:

```py
        # Attempt to parse potential thumbnail
        print_thumnail = None
        if event_id == "printing_started" and self.get_gcode_format(data["path"]):
            self._logger.info("Print appears to have a thumbnail, trying to parse")
            print_thumbnail = self.get_thumbnail_cura(data["path"])
```